packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -1511,17 +1511,27 @@ const DeleteResponseCodes = (deleteOp, _opts, ctx) => {
     return errors;
 };
 
-const getCollectionOnlyHasValueAndNextLink = (properties, _opts, ctx) => {
-    if (!properties || typeof properties !== "object") {
-        return [];
-    }
-    const keys = Object.keys(properties);
-    if (keys.length != 2 || !keys.includes("value") || !keys.includes("nextLink")) {
-        return [
-            {
-                message: "Get endpoints for collections of resources must only have the `value` and `nextLink` properties in their model.",
-            },
-        ];
+const getCollectionOnlyHasValueAndNextLink = (prop, _opts, ctx) => {
+    for (let path of ctx.path) {
+        if (path.includes(".")) {
+            let splitNamespace = path.split(".");
+            if (path.includes("/")) {
+                let segments = splitNamespace[1].split("/");
+                if (segments.length % 2 !== 0) {
+                    return [];
+                }
```

Comment:
add a comment at the top of this code block to indicate why this logic is being added.